frontend/src/navbar/navbar.html

```diff
@@ -12,11 +12,11 @@
       <a
         href="#/"
         class="inline-flex items-center px-1 pt-1 border-b-2 text-sm font-medium"
-        :class="routeName === 'root' ? 'text-gray-900 border-ultramarine-500' : 'border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-700'">Documents</a>
+        :class="!dashboardView ? 'text-gray-900 border-ultramarine-500' : 'border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-700'">Documents</a>
```

Comment:
This won't work when we have more nav items (I currently have a new "shell" tab WIP here: https://github.com/mongoosejs/studio/tree/vkarpov15/shell)

Reply:
I figured I would adjust it if we ever got to that point. I'll make the necessary changes